src/Collection/CollectionBuilder.php

```diff
@@ -71,6 +71,21 @@ public function __construct($commandFile)
     }
 
     /**
+     * @param int $logLevel
+     *
+     * @return $this
+     */
+    public function setLogLevel($logLevel)
```

Comment:
Hm. I tried removing this from Collection Builder but it breaks the functionality. It doesn't seem as though the property percolates into the required task object without this.